File: api/FinanceApi/FinanceApi/Repositories/EF Models/vExpense.cs
using System.ComponentModel.DataAnnotations;

namespace FinanceApi.Repositories.EF_Models
{
    public class vExpense
    {
        public vExpense()
        {
            ExpenseID = 0;
            ExpenseTypeID = 0;
            PaymentTypeID = 0;
            PaymentTypeCategoryID = 0;
            IsIncome = false;
            IsInvestment = false;
            ExpenseDescription = string.Empty;
            ExpenseAmount = 0;
            ExpenseDate = new DateTime(1, 1, 1);
            LastUpdated = new DateTime(1, 1, 1);
        }
        [Key]
        public int ExpenseID { get; set; }
        public bool IsIncome { get; set; }
        public bool IsInvestment { get; set; }
        public int ExpenseTypeID { get; set; }
        public int PaymentTypeID { get; set; }
        public int PaymentTypeCategoryID { get; set; }
        public string ExpenseDescription { get; set; }
        public DateTime ExpenseDate { get; set; }
        public DateTime LastUpdated { get; set; }
        public decimal ExpenseAmount { get; set; }
    }
}

File: api/Node/finance-api/src/entity/v_expense.ts
import { ViewEntity, ViewColumn } from "typeorm";

@ViewEntity({
  name: "v_expense",
  expression: "",
  synchronize: false,
})
export class v_expense {
  @ViewColumn({ name: "expense_id" })
  expenseID?: number;

  @ViewColumn({ name: "expense_type_id" })
  expenseTypeID?: number;
  
  @ViewColumn({ name: "payment_type_id" })
  paymentTypeID?: number;

  @ViewColumn({ name: "payment_type_category_id" })
  paymentTypeCategoryID?: number;

  @ViewColumn({ name: "is_income" })
  isIncome?: boolean;

  @ViewColumn({ name: "is_investment" })
  isInvestment?: boolean;

  @ViewColumn({ name: "expense_description" })
  expenseDescription?: string;

  @ViewColumn({ name: "expense_amount" })
  expenseAmount?: number;

  @ViewColumn({ name: "expense_date" })
  expenseDate?: Date;

//   @ViewColumn({ name: "" })
//   LastUpdated?: Date;
}